stored_requests/events/files/files.go

```diff
@@ -0,0 +1,47 @@
+package files
+
+import (
+	"context"
+
+	"github.com/golang/glog"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/stored_requests"
+	"github.com/prebid/prebid-server/stored_requests/backends/file_fetcher"
+	"github.com/prebid/prebid-server/stored_requests/events"
+)
+
+type fileEventProducer struct {
+	invalidations chan events.Invalidation
+	saves         chan events.Save
+}
+
+func (f *fileEventProducer) Saves() <-chan events.Save {
+	return f.saves
+}
+func (f *fileEventProducer) Invalidations() <-chan events.Invalidation {
+	return f.invalidations
+}
+
+// NewFilesLoader returns an EventProducer preloaded with all the stored reqs+imps
+func NewFilesLoader(cfg config.FileFetcherConfig) events.EventProducer {
+	fp := &fileEventProducer{
+		saves:         make(chan events.Save, 1),
```

Comment:
I've seen this done elsewhere in our code, but why are we using a  buffered channel for the save events?

Reply:
The capacity is needed to preload the data set in the EventProducer before returning, so it is a little closer to the original behavior which also did a blocking preload of all files.